src/main/java/hudson/plugins/fitnesse/FitnesseResults.java

```diff
@@ -38,7 +38,7 @@
 	private FitnesseResults parent;
 	private List<FitnesseResults> details = new ArrayList<FitnesseResults>();
 	private transient Run<?,?> owner;
-	private TaskListener listener;
+	private transient TaskListener listener;
```

Comment:
Generally need to either restore in `readResolve`, or stop making it an instance field altogether (use a local variable), or access only via a getter which sets it on demand.

Reply:
Right. It would be also reasonable to rename the field at least so that it does not cause deserialization warnings for obsolete data